Atomic/mount.py

```diff
@@ -738,6 +738,31 @@ def _clean_temp_container_by_path(self, path):
             self.d.remove_container(short_cid)
         self._clean_tmp_image()
 
+
+class DockerMountContextManager(object):
```

Comment:
Because it's specifically targeted to API of `DockerMount` class. Shall I make it work also for `Mount` class?